packages/11ty/_lib/chalk/index.js

```diff
@@ -0,0 +1,22 @@
+const chalk = require('chalk')
+
+module.exports = function(context='') {
```

Comment:
We might rename this argument to `prefix` so that it is not confused with the Eleventy rendering `context`.